comet/manager.py

```diff
@@ -163,11 +172,11 @@ def register_start(self, start_time, version, config=None):
             "time": start_time.strftime(TIMESTAMP_FORMAT),
             "version": version,
         }
-        if config:
-            data["config_state"] = self.states[self.config_state].to_dict()
-        state = State(data, "start_{}".format(name))
+        if config and not register_datasets:
+            data["config_state"] = self.config_state.to_dict()
```

Comment:
Can you please explain what impact it has to add `config_state` to the `data` attribute?

Reply:
I injects the config into the start state.